src/features/metadata-based-view/MetadataBasedItemList.scss

```diff
@@ -1,5 +1,8 @@
 @import '../../styles/variables';
 
+$inset-shadow-left: -4px 0 2px -2px $bdl-gray-10 inset;
+$inset-shadow-right: 3px 0 3px -2px $bdl-gray-10 inset;
```

Comment:
Local variable names should follow this format: https://confluence.inside-box.net/display/ETO/CSS+Class+Naming#CSSClassNaming-SCSSVariables. So, these would be `bdl-MetadataBasedItemList-insetShadowLeft` and `bdl-MetadataBasedItemList-insetShadowRight`.